core/computed/page-dependency-graph.js

```diff
@@ -20,16 +19,12 @@ class PageDependencyGraph {
    * @return {Promise<Node>}
    */
   static async compute_(data, context) {
-    const {trace, devtoolsLog} = data;
+    const {trace, devtoolsLog, URL} = data;
     const [processedTrace, networkRecords] = await Promise.all([
       ProcessedTrace.request(trace, context),
       NetworkRecords.request(devtoolsLog, context),
     ]);
 
-    // COMPAT: Backport for pre-10.0 clients that don't pass the URL artifact here (e.g. pubads).
-    // Calculates the URL artifact from the processed trace and DT log.
-    const URL = data.URL || await DocumentUrls.request(data, context);
```

Comment:
This looks like the only usage of `DocumentUrls`, we can probably just remove it.